_departmentpeople/valiagaz

```diff
@@ -0,0 +1,23 @@
+---
+UVA_id: eg4va
+lastname: Gazaki
+name: Evangelia
+general_position: faculty
+position: Assistant Professor
+office: 221 Kerchof Hall
+phone: 434-924-4930
+email: eg4va@virginia.edu
+image: 
+personal_page: http://people.virginia.edu/~eg4va/
+areas:
+  - Algebra&nbsp;$$\cup$$&nbsp;Representation Theory
+  - Geometry&nbsp;$$\cup$$&nbsp;Topology
+grad_general_areas:
+  - algebra
+grad_special_areas:
+  - Number Theory
```

Comment:
This is used to form people lists here https://math.virginia.edu/graduate/research-areas/ and Number Theory is not among the areas. The whole lists are also somewhat outdated. Consider changing this to one of the existing areas (see source of https://math.virginia.edu/graduate/research-areas/ on github); or ignore